install/kubernetes/helm/istio/values.yaml

```diff
@@ -407,6 +408,7 @@ tracing:
     externalPort: 9411
     internalPort: 9411
     uiPort: 16686
+    jaegerLoadBalancerPort: 12346
```

Comment:
Don't think this should refer to jaeger, what about `externalUiPort` and rename the property above (i.e. uiPort) as `internalUiPort`?

Reply:
+1, thanks @objectiser